src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml

```diff
@@ -0,0 +1,159 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: xx.1.1. [TC-OPSTATE-1.1] Global attributes with server as DUT
+
+PICS:
+    - OPSTATE.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Operational State"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision attribute from the DUT"
+      PICS: OPSTATE.S.Afffd
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS:
+          OPSTATE.S.Afffc
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "TH reads the AttributeList attribute from the DUT"
+      PICS: OPSTATE.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0,
+                      1,
+                      3,
+                      4,
+                      5,
+                      65528,
+                      65529,
+                      65530,
+                      65531,
+                      65532,
+                      65533,
+                  ]
+
+    - label:
+          "TH reads from the DUT the optional attribute(CountdownTime) in the AttributeList 
+          from the DUT"
+      PICS: OPSTATE.S.A0002 && OPSTATE.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [2]
+
+    - label:
+          "TH reads from the DUT the EventList attribute."
+      PICS: OPSTATE.S.Afffa
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+              contains: [0x00]
+              
+    - label:
+          "TH reads from the DUT the optional event(OperationCompletion) in EventList."
+      PICS: OPSTATE.S.E01 && OPSTATE.S.Afffa
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+              contains: [0x01]
+
+    - label: "Read the optional command(Start) in AcceptedCommandList"
+      PICS: OPSTATE.S.C00 && OPSTATE.S.Afff9
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                            0,
+                            1
```

Comment:
So supporting Pause implies supporting Stop?  Spec says Pause implies Resume, but nothing about Stop....

Or is there just confusion here because someone thought id 0 is Start?  Start is 3.